connectors/sources/github.py

```diff
@@ -1287,39 +1288,32 @@ async def _fetch_files(self, repo_name, default_branch):
             )
 
             for repo_object in file_tree.get("tree", []):
-                last_commit_timestamp = await self._fetch_last_commit_timestamp(
```

Comment:
Should we be fetching the last_commit_timestamp for the markdown files? I see that we weren't before when type was `BLOB`, but I don't understand why.

Reply:
Earlier, we were fetching `last_commit_timestamp` for all the documents as we were indexing all the files. but now we are indexing markdown files only, so we are fetching `last_commit_timestamp` for markdown files only in L1299.

Reply:
omg Idk how I missed that. Thanks.